src/adagrams.js

```diff
@@ -1,15 +1,181 @@
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const getRandomIdx = () => {
+  const NUM_OF_LETTERS_IN_LETTER_BANK = 96
+  return Math.floor(Math.random() * NUM_OF_LETTERS_IN_LETTER_BANK)
+};
+
+const getLetterBank = (LETTER_POOL)=> {
+  let letterBank = []
+  for(let letter in LETTER_POOL){
+    const freq = LETTER_POOL[letter]
+	    letterBank.push(...letter.repeat(freq).split(''))
```

Comment:
Another way to write this code would be to use `Object.entries`
```suggestion
for (const [letter, value] of Object.entries(LETTER_POOL)){
    for (let i = 0; i < value.qty; i++){
      letterBank.push(letter);
```
We can directly use the value from `LETTER_POOL` to loop the correct number of times to add the correct number of letters, so we wouldn't need `repeat` or `split`.